pegasus/sites.v3/code.org/views/theme_common_head_after.haml

```diff
@@ -8,6 +8,8 @@
 
 %script=view 'google_analytics.js', user_type: user_type
 
+-# webpack-runtime.js must appear exactly once on any page containing webpack entries
+%script{src: minifiable_asset_path('js/webpack-runtime.js')}
```

Comment:
@breville , are you aware of any parts of the pegasus sites that might not use theme_common_head_after? It would be good to know of any since webpack js entry points won't work there.

Reply:
On a hunch, I checked https://code.org/curriculum/csp and it appears to not use that file.

Reply:
Oh, that worked non-production environments, but not on production itself. https://code.org/curriculum/unplugged does not use theme_common_head_after.